app/views/appearances/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :page_header do %>
+  <h1><span class="fa fa-paint-brush"></span> <%= t(:'sufia.admin.sidebar.appearance') %></h1>
+<% end %>
+
+<div class="row">
+  <div class="col-md-12">
+    <div class="panel panel-default">
+      <%= form_for(@site, url: site_appearances_path) do |f| %>
+        <div class="panel-heading">
+          <h3 class="panel-title">Site Appearance</h3>
+        </div>
+        <div class="panel-body">
+          <% if @site.errors.any? %>
+              <div id="error_explanation">
+                  <h2><%= pluralize(@site.errors.count, "error") %> prohibited this site from being saved:</h2>
+
+                  <ul>
+                      <% @site.errors.full_messages.each do |message| %>
+                          <li><%= message %></li>
+                      <% end %>
+                  </ul>
+              </div>
+          <% end %>
+          <div class="field form-group">
+              <%= f.label :banner_image, '<span class="fa fa-camera"></span> Change site banner image'.html_safe %><br>
```

Comment:
I take it all back, @cbeer. 👍 Thanks for the hint on `simple_form`.  I hadn't used that before.. it's rather cool.  Updating this to page to just use `simple_form` which vastly simplifies the code.